GameServerLib/GameObjects/Other/CollisionHandler.cs

```diff
@@ -106,7 +106,7 @@ public void Update()
                     continue;
                 }
 
-                if (!_map.NavigationGrid.IsWalkable(obj.X, obj.Y))
+                if (!_map.NavigationGrid.IsWalkable(obj.Position.X, obj.Position.Y))
```

Comment:
IsWalkable still gets X,Y

Reply:
AttackableUnits can move, but can't target, hence the separation between dashing to a location vs a target object.

Reply:
I'm kind of out of the loop, but IIRC an inhibitor is an ``AttackableUnit``, so I'm not sure why an ``AttackableUnit`` should be able to move...

Reply:
Although AttackableUnits can move, this doesn't mean they will move by themselves. The only instance where they will move is when an attack forces them to dash. Buildings of course will override this.